tripkituisample/src/main/AndroidManifest.xml

```diff
@@ -22,14 +21,15 @@
         <!-- replace with ${google_map_key} to get key from local.properties -->
         <meta-data
             android:name="com.google.android.geo.API_KEY"
-            android:value="GOOGLE_MAP_API_KEY_HERE" />
+            android:value="AIzaSyDDG_4m7xVSCA4gZj-RlgBVFCnhoPMh7oo" />
```

Comment:
Quick question, what's the reason behind hardcoding this string? @MichaelReyes

Reply:
Nyay, this should not be included in the PR. let me revert this one. Thanks @sg-jsonjuliane